### PR TITLE
Remove cython from build

### PR DESCRIPTION
**Pull request by randyzwitch**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by randyzwitch** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1000
+  number: 0
```
Comment (xhochy) [inline recipe/meta.yaml]: This should really be 1000  until the compiler migration is done on Tuesday

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by xhochy** [comment]
Seems like the rerender is not adding the new compilers, I'll have a look tomorrow morning

File: recipe/meta.yaml (hunk 1 above)
Comment (randyzwitch) [reply recipe/meta.yaml]: Pushed another commit to put it back to 1000

**Pull request closed without merging**